python/tchannel/scheme.py

```diff
@@ -77,3 +76,22 @@ def deserialize_body(self, obj):
 
     def serialize_body(self, obj):
         return json.dumps(obj)
+
+
+class RawArgScheme(ArgScheme):
+    _type = "raw"
+
+    def type(self):
+        return self._type
```

Comment:
I want to make it as a required field. If user creates customerized the argscheme subclass, make sure they have type in it.